product_docs/docs/pgd/5/durability/comparing.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Comparing Durability Options
+navTitle: Comparing Durability Options
+deepToC: true
+---
+
+## Comparison
+
+Most options for synchronous replication available to PGD allow for different
+levels of synchronization, offering different tradeoffs between performance and
+protection against node or network outages.
+
+The following list of [confirmation levels](commit-scope-rules#the-confirmation-level) explains
+what a user should expect to see when that confirmation level in in effect and how that can effect performance, durability, and consistency.
+
+### ON RECEIVED
+
+**Expect**: The peer node has received the changes. Nothing has been updated in
+the peer nodes tuple store or written to storage.
+
+Confirmation on reception means that the peer operating normally can
+eventually apply the transaction without requiring any further
+communication, even in the face of a full or partial network
+outage. A crash of a peer node might still require retransmission of
+the transaction, as this confirmation doesn't involve persistent
+storage. 
+
+**For**: Good performance.
+
+**Against**: Less robust when a crash occurs.
+
+### ON REPLICATED
+
+**Expect**: The peer node has received the changes and applied them to the tuple
+store. The changes have not been persisted to storage.
+
+Confirmation on replication means the peer has received and applied the changes, but 
+has yet to persist them to disk. 
+
+**For**: A better checkpoint than received with less performance.
+
+**Against**: Less robust when a crash occurs.
```

Comment:
The difference between received and replicated is that replicated data survives PG crash, but not OS crash, received does not survive even PG crash.

Reply:
I think we need to do better to document that here